Components/Avatar/avatar.css

```diff
@@ -0,0 +1,33 @@
+* {
+    box-sizing: border-box;
+    font-family: sans-serif;
+}
+
+:root {
+    --avatar-size: 2em;
+    --avatar-border: none;
+}
+
+.avatar {
+    width: var(--avatar-size);
+    border: var(--avatar-border);
+    border-radius: 50%;
+    margin: 2.4rem 0.5rem;
+    margin-top: 1.2rem;
```

Comment:
You can use utility class for margins, as it will provide you with greater flexibility.

Reply:
Ok as i was short on time, and a bit slow i did not do this, would definitely try to implement it later. Thanks!